docs/internals/layout_in_storage.rst

```diff
@@ -240,66 +240,66 @@ value and reference types, types that are encoded packed, and nested types.
 
 .. code:: json
 
-    "storageLayout": {
```

Comment:
The issue was that this wasn't well-defined JSON.

This is the output of `solc --storage-layout` of the above code. It doesn't have "storageLayout" key, so, skipping it.